app/controllers/api/v2/report_templates_controller.rb

```diff
@@ -0,0 +1,136 @@
+module Api
+  module V2
+    class ReportTemplatesController < V2::BaseController
+      include Foreman::Controller::Parameters::ReportTemplate
+      include Foreman::Controller::TemplateImport
+      include UnattendedHelper
+      include Foreman::Controller::ProvisioningTemplates
```

Comment:
no longer needed, I know that without this, renderer used to fail since it tried to call `load_vars_from_template`, which is why we also had this method empty here, it works fine after refactoring